xml.go

```diff
@@ -71,6 +98,7 @@ type MetaData struct {
 	MasterKeyChangeRec         int64          `xml:"MasterKeyChangeRec"`
 	MasterKeyChangeForce       int64          `xml:"MasterKeyChangeForce"`
 	MemoryProtection           MemProtection  `xml:"MemoryProtection"`
+	CustomIcons                []CustomIcon   `xml:"CustomIcons>Icon"`
```

Comment:
If no custom icons are defined, this is not part of the xml when it comes from KDBX and therefore should probably be optional